spec/models/sales_log_spec.rb

```diff
@@ -564,37 +564,6 @@ def check_property_postcode_fields
       expect(record_from_db["la"]).to eq("E08000003")
     end
 
-    context "with 23/24 logs" do
```

Comment:
This became remarkably painful to try and fix, and becomes irrelevant so didn't seem worth too much effort.